dataset_generator/data/README.md

```diff
@@ -4,6 +4,45 @@ Data in this folder generated as part of unknown borrowers research.
 
 - exceptional members
 - exceptional books
-- exceptioal events
+- exceptional events
 - partial borrowers
-- genre/subject data
\ No newline at end of file
+- genre/subject data
+- long borrow overrides: corrected information borrow events that are incorrectly entered with durations longer than a year in the 1.2 events dataset
```

Comment:
Oof didn't know about this

Reply:
yeah, I just discovered them when I was writing up the case studies for the members we are making recommendations for – Colens had one weird outlier for borrowing duration (only one borrow over a year), and when I looked at the card it seemed like a data entry error — should have been a few days later instead of a year later. I did a quick colab to flag all the borrows over a year and Josh reviewed and correct them, but unfortunately I didn't discover it early enough to include in the official 1.2 version.

There aren't a ton of them, but I figured any duration that should be days and looks like years is going to mess up some of our calculations.